docs/authentication/social-connections/apple.mdx

```diff
@@ -73,6 +79,8 @@ In _production instances_, you must provide custom credentials, which includes g
   1. Select **Continue**. You'll be taken to the confirmation page.
   1. Select **Save**.
 
+  The **Apple Services ID** is required for web based OAuth flows and recommended for native app flows.
```

Comment:
I had moved these to the top of the steps, because it's important for a user to see if they even need to follow this step or not. Is there a reason you moved them back to the bottom?

Reply:
:confused: No reason at all, I didn't even realize it. Sorry!

Moved all similar callouts at the top of each step description.

Reply:
no worries ! thank you :)